mods/cnc/languages/rules/en.ftl

```diff
@@ -397,16 +394,16 @@ actor-hq =
 
 actor-fix =
    .name = Repair Facility
-   .description = Repairs vehicles
+   .description = Repairs vehicles.
    .encyclopedia = Repairs vehicles. The repair command can be used on units to send them to the nearest repair facility.
 
 actor-eye =
    .name = Advanced Communications Center
    .description = Provides radar and Orbital Ion Cannon support power.
     Unlocks Mammoth Tank and Commando.
-    Requires power to operate.
+    Requires power to operate
    .ioncannonpower-name = Ion Cannon
-   .ioncannonpower-description = Initiate an Ion Cannon strike.
+   .ioncannonpower-description = Initiates an Ion Cannon strike.
```

Comment:
If we're changing this, then I would argue "Launch a tactical nuclear warhead" should be "Launches" to match - for `actor-tmpl` underneath.